decimal.go

```diff
@@ -614,20 +614,16 @@ func (d Decimal) Mod(d2 Decimal) Decimal {
 	return d.Sub(d2.Mul(quo))
 }
 
-// Pow returns d to the power d2
 func (d Decimal) Pow(d2 Decimal) Decimal {
```

Comment:
Could we add a comment that would explain the given solution, just for future devs who will work on improvements or modification of this method?